services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/structure/file/FileOnDisk.java

```diff
@@ -60,6 +60,10 @@ public void setBytes(byte[] bytes) {
    * Does nothing.
    */
   @Override
-  public void prepare(ImmutableStack<Object> indices) {
+  public void prepare(ImmutableStack<Object> indices){
```

Comment:
Please re-add the single space before the curly bracket

Reply:
Single space added. Change pushed.